AdminUi/src/AdminUi/ClientApp/src/app/components/client/client-edit/client-edit.component.ts

```diff
@@ -183,23 +176,22 @@ export class ClientEditComponent {
         });
     }
 
-    public setDefaultTier(): boolean {
-        if (!this.client.defaultTier) {
-            const basicTier = this.tierList.find((tier) => tier.name === "Basic");
-            if (basicTier) {
-                this.client.defaultTier = basicTier.id;
-            } else {
-                this.snackBar.open("Basic Tier not found", "Dismiss", {
-                    verticalPosition: "top",
-                    horizontalPosition: "center"
-                });
-                return false;
-            }
-        }
-        return true;
-    }
-
     public togglePasswordVisibility(): void {
         this.showPassword = !this.showPassword;
     }
+
+    private getDefaultTier(): string {
+        const basicTier = this.tierList.find((tier) => tier.name === "Basic");
+        if (basicTier) {
+            return basicTier.id;
+        }
+
+        this.snackBar.open("Basic Tier not found", "Dismiss", {
+            verticalPosition: "top",
+            horizontalPosition: "center"
+        });
+
+        this.disabled = true;
```

Comment:
The GetTiers endpoint should always return a list that includes the Basic Tier. If it couldn't find the Basic Tier in that list, then there was an error somewhere, so I decided it would be best to disable the page so the user doesn't waste their time trying to fill the missing fields.
After the changes above this is no longer relevant, and has been removed.